Specs/DateTime/English/TimePeriodExtractor.json

```diff
@@ -546,4 +546,40 @@
     }
   ]
 }
+,
+{
+  "Input": "set up meeting from 1:30 to 3:30",
```

Comment:
Why is this not supported in JS? There was a bug in the old code for such cases?